content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -45,64 +45,35 @@ imply any preferential status.
+Project [Calico](https://projectcalico.docs.tigera.io/about/about-calico/) is an open-source networking solution which provides a [CNI plugin](https://projectcalico.docs.tigera.io/networking/determine-best-networking#cni-plugins) and various other networking plugins compatible with Kubernetes.
 
 ### Cilium
 
-[Cilium](https://github.com/cilium/cilium) is open source software for
+Project [Cilium](https://github.com/cilium/cilium) is an open-source solution for
 providing and transparently securing network connectivity between application
-containers. Cilium is L7/HTTP aware and can enforce network policies on L3-L7
-using an identity based security model that is decoupled from network
-addressing, and it can be used in combination with other CNI plugins.
-
-### CNI-Genie from Huawei
-
-[CNI-Genie](https://github.com/cni-genie/CNI-Genie) is a CNI plugin that enables Kubernetes to [simultaneously have access to different implementations](https://github.com/cni-genie/CNI-Genie/blob/master/docs/multiple-cni-plugins/README.md#what-cni-genie-feature-1-multiple-cni-plugins-enables) of the [Kubernetes network model](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model) in runtime. This includes any implementation that runs as a [CNI plugin](https://github.com/containernetworking/cni#3rd-party-plugins), such as [Flannel](https://github.com/flannel-io/flannel#flannel), [Calico](https://projectcalico.docs.tigera.io/about/about-calico/), [Weave-net](https://www.weave.works/oss/net/).
-
-CNI-Genie also supports [assigning multiple IP addresses to a pod](https://github.com/cni-genie/CNI-Genie/blob/master/docs/multiple-ips/README.md#feature-2-extension-cni-genie-multiple-ip-addresses-per-pod), each from a different CNI plugin.
+containers.
 
 ### cni-ipvlan-vpc-k8s
-[cni-ipvlan-vpc-k8s](https://github.com/lyft/cni-ipvlan-vpc-k8s) contains a set
-of CNI and IPAM plugins to provide a simple, host-local, low latency, high
-throughput, and compliant networking stack for Kubernetes within Amazon Virtual
-Private Cloud (VPC) environments by making use of Amazon Elastic Network
-Interfaces (ENI) and binding AWS-managed IPs into Pods using the Linux kernel's
-IPvlan driver in L2 mode.
-
-The plugins are designed to be straightforward to configure and deploy within a
-VPC. Kubelets boot and then self-configure and scale their IP usage as needed
-without requiring the often recommended complexities of administering overlay
-networks, BGP, disabling source/destination checks, or adjusting VPC route
-tables to provide per-instance subnets to each host (which is limited to 50-100
-entries per VPC). In short, cni-ipvlan-vpc-k8s significantly reduces the
-network complexity required to deploy Kubernetes at scale within AWS.
+[cni-ipvlan-vpc-k8s](https://github.com/lyft/cni-ipvlan-vpc-k8s) is a CNI plugins that aims to reduce the network complexity required to deploy Kubernetes at scale within AWS.
```

Comment:
I think it is okay to provide a brief, high-level overview of a project/technology like this, provided that the text contains no vendor pitches or things like that. A reader may decide whether to click the link based on this short introduction.